frontend/public/locales/en/translation.json

```diff
@@ -147,6 +147,102 @@
     "more": "More",
     "privacyPolicy": "Privacy policy"
   },
+  "terms": {
+    "termsOfService": "Terms of service",
+    "acceptableUse": {
+      "acceptableUse": "C. Acceptable Use",
+      "shortVersion": "While using the Service, you must follow the terms of this section, which include not harming other Users, not disturbing the Service, and having an authentic activity.",
+      "titles": {
+        "1ComplianceWithLawsAndRegulation": "1. Compliance with Laws and Regulations",
+        "2UserSafety": "2. User Safety",
+        "3InauthenticActivityAndSpam": "3. Inauthentic Activity and Spam"
+      },
+      "paragraphs": {
+        "complianceWithLawsAndRegulation": "You are responsible for using the Service in compliance with all applicable laws, regulations, and these Acceptable Use policies. ",
+        "userSafetyIntro": "We do not allow activities on the Platform, including using a public username, that:",
+        "isUnlawlfulOrPromoteUnlawful": "are unlawful or promotes unlawful activities;",
+        "isFalseInaccurateOrDeceptive": "are false, inaccurate, or intentionally deceptive information and likely to adversely affect the public interest (including health, safety, election integrity, and civic participation);",
+        "orGoesAgainstOurCodeOfConduct": "or go against our <2>Code of Conduct</2>.",
+        "inauthenticActivityAndSpamIntro": "We do not allow activity the Platform that is: inauthentic interactions, such as fake accounts and automated inauthentic activity; or automated excessive bulk activity and coordinated inauthentic activity."
```

Comment:
Suggestion to make the definition of "authentic behavior" a bit more precise:

Users are expected to have an authentic behaviour on the Platform: this means expressing their sincere opinions when submitting data, without any intent to deceive, manipulate, or misrepresent information. We do not allow inauthentic interactions the Platform; in particular using fake accounts, automating or coordinating inauthentic activity is not allowed.